pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -85987,7 +85987,7 @@ self: {
 
   "fltkhs" = callPackage
     ({ mkDerivation, base, bytestring, c2hs, Cabal, directory, filepath
-     , fltk14, libGLU_combined, mtl, OpenGLRaw, parsec, pkgconfig, text
```

Comment:
This file is generated automatically.  If this is merged in as-is, it will be overwritten to use `libGLU_combined` again next time it is automatically generated.

Could you send a PR to [`cabal2nix`](https://github.com/NixOS/cabal2nix) teaching it to not use `libGLU_combined`?

I believe the line that needs changing is as follows:

https://github.com/NixOS/cabal2nix/blob/d19721cf8b5c0e712638e41a3d187fef31a21680/src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs#L92

If you can't figure out the right syntax for this, then please open a PR with what you have, ping me, and I'll take a look at it and get it into shape.